modules/ROOT/pages/_partials/commons/common-new-logging-api.adoc

```diff
@@ -0,0 +1,634 @@
+// {param-module} grabs the correct platform name from the related _set_page_context_for_[platform] 
+== Upgrading to the New CBL Logging API
+
+CAUTION: Use of the deprecated and new Logging API at the same time is not supported.
+
+You can find information about the new Couchbase Lite Logging API introduced in Couchbase Lite 3.2.2.
+
+ifndef::is-c[]
+
+For information about the now deprecated earlier version of the Logging API, see xref:3.1@couchbase-lite:{param-module}:troubleshooting-logs.adoc[]
+
+endif::is-c[]
+
+== LogSinks
+
+Couchbase Lite 3.2.2 introduces a new Logging API.
+
+The first thing to note is that the three destinations for logs have been renamed as `LogSinks`, in keeping with common source/sink terminology. 
+
+The `FileLogSink`, the `ConsoleLogSink` and the `CustomLogSink` are all installed in an instance of `LogSinks`. 
+
+All three log sinks are initialized by default. 
```

Comment:
It seems this could give an impression that all log sinks are enabled. I think it's more accurate to say that only ConsoleLogSink is enabled for all logging domains at the warning log level by default.

Reply:
I second this. Moreover, I think it's "more correct" to maybe say that "all are to be installed in the LogSinks instance". There is only 1 instance of LogSinks and is available to the user right away.